docs/user-guide/glossary/edit-glossaries.md

```diff
@@ -1,6 +1,28 @@
 ### Prerequisite
 See [glossary roles and permission](/user-guide/glossary/glossary-roles-permissions) for permission setup.
 
+### From Glossary page
+1. Click `Glossary` menu.
+<figure>
+![Glossary page](/images/glossary-page.png)
+</figure>
+1. Go to entry you wish to edit, fill in part of speech or description.
+<figure>
+![Glossary edit](/images/glossary-edit.png)
+</figure>
+1. Click `Update` to save your changes, or `Cancel` to revert your changes, `Delete` to delete this entry.
```

Comment:
Flip these so that the instruction comes after what it does. e.g. `To save your changes, click Update...`